### PR TITLE
Add privacy considerations related to legal processes and coercion

### PR DESCRIPTION
**Pull request by msporny**
This PR is an attempt to address issue #1502 by adding a few more privacy considerations related to legal processes and coercion.

/cc @zoracon

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1504.html" title="Last updated on Jun 26, 2024, 11:16 PM UTC (7072c8c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/1504/0a82f17...7072c8c.html" title="Last updated on Jun 26, 2024, 11:16 PM UTC (7072c8c)">Diff</a>

### CODE REVIEW

**Review by decentralgabe — APPROVED** [review]

**Review by iherman — APPROVED** [review]

**Review by dlongley — APPROVED** [review]

**Review by longpd — COMMENTED** [review]
The phrasing "such that a [=holder=] **can push back on** the over-collection of information that is unnecessary for the transaction." I presume means the holder can decline to share specific information requested.  Is that what this implies? If so, then saying that explicitly would be better.

**Comment by msporny** [comment]
> I presume means the holder can decline to share specific information requested. Is that what this implies? If so, then saying that explicitly would be better.

Yes, that's what was meant, your text is more clear. I'll update the PR to use your text, thanks @longpd !

**Review by brentzundel — APPROVED** [review]

**Review by zoracon — APPROVED** [review]
The phrasing and changes overall address my initial concerns about addressing law enforcement, coercion, and the unreliability of private browsing.

**Review by David-Chadwick — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -5425,6 +5425,10 @@ <h3>The Principle of Data Minimization</h3>
 Implementers of software used by [=holders=] are urged to disclose what
 information is being requested by a [=verifier=], such that a [=holder=] can
 push back on the over-collection of information that is unnecessary for the
```
Comment (TallTed) [inline index.html:5427]: @longpd's suggested revision —
```suggestion
information is being requested by a [=verifier=], such that a [=holder=] can
decline to share specific requested information that is unnecessary for the
```

File: index.html [hunk 2]
```diff
@@ -5764,6 +5768,23 @@ <h3>Usage Patterns</h3>
...
+It is possible, through legal processes, for private information to be compelled
+to be disclosed to authorities, such as law enforcement, by [=issuers=],
+[=holders=], and/or [=verifiers=]. It is also possible for the same private
+information to be accidentally disclosed to an unauthorized party through a
+software bug or security failure. Authors of legal processes and compliance
+regimes are advised to draft guidelines that notify the [=subjects=] involved
+when their private information is purposefully or accidentally disclosed to a
+third party. Providers of software services are advised to be transparent about
+who and under what circumstances that private information might be shared with a
+third party.
```
Comment (TallTed) [inline index.html:5784]:
```suggestion
It is possible, through legal processes, for [=issuers=], [=holders=], and/or
[=verifiers=] to be compelled to disclose private information to authorities,
such as law enforcement. It is also possible for the same private
information to be accidentally disclosed to an unauthorized party through a
software bug or security failure. Authors of legal processes and compliance
regimes are advised to draft guidelines that notify the [=subjects=] involved
when their private information is purposefully or accidentally disclosed to a
third party. Providers of software services are advised to be transparent about
known circumstances that might cause such private information to be shared with
a third party, and about the identity of any such third party.
```

**Review by TallTed — CHANGES_REQUESTED** [review]

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**